test_unstructured/partition/html/test_parser.py

```diff
@@ -0,0 +1,600 @@
+# pyright: reportPrivateUsage=false
+# pyright: reportUnknownArgumentType=false
+
+"""Test suite for `unstructured.partition.html.parser` module."""
+
+from __future__ import annotations
+
+from collections import deque
+
+import pytest
+from lxml import etree
```

Comment:
I'm getting warnings from Pylance about methods of etree being partially unknown. Do we need to add a stub file for lxml?

Reply:
Hmm, that's a good point. I have `types_lxml` installed locally so I'm not getting those, but it would be better for me to just add the few stubs we need into `/typings`. I'll do that.

Reply:
k, done.